resources/web/streaming_viewer/index.html

```diff
@@ -1,74 +1,34 @@
-    <script src='https://cdn.jsdelivr.net/ace/1.2.6/min/ace.js'></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=MML_HTMLorMML"></script>
-    <script src='../wwi/dependencies/three.min.js'></script>
-    <script src='../wwi/dependencies/loaders/RGBELoader.js'></script>
-    <script src='../wwi/dependencies/pmrem/PMREMCubeUVPacker.js'></script>
-    <script src='../wwi/dependencies/pmrem/PMREMGenerator.js'></script>
-    <script src='../wwi/dependencies/shaders/CopyShader.js'></script>
-    <script src='../wwi/dependencies/shaders/FXAAShader.js'></script>
-    <script src='../wwi/dependencies/postprocessing/EffectComposer.js'></script>
-    <script src='../wwi/dependencies/postprocessing/RenderPass.js'></script>
-    <script src='../wwi/dependencies/postprocessing/ShaderPass.js'></script>
-    <script src='../wwi/shaders/hdr_resolve.js'></script>
-    <script src='../wwi/shaders/bright_pass.js'></script>
-    <script src='../wwi/shaders/blend_bloom.js'></script>
-    <script src='../wwi/shaders/gaussian_blur_13_tap.js'></script>
-    <script src='../wwi/passes/bloom.js'></script>
-    <script src='../wwi/gpu_picker.js'></script>
-    <script src='../wwi/util.js'></script>
-    <script src='../wwi/system_info.js'></script>
-    <script src='../wwi/dialog_window.js'></script>
-    <script src='../wwi/console.js'></script>
-    <script src='../wwi/help_window.js'></script>
-    <script src='../wwi/robot_window.js'></script>
-    <script src='../wwi/toolbar.js'></script>
-    <script src='../wwi/editor.js'></script>
-    <script src='../wwi/context_menu.js'></script>
-    <script src='../wwi/viewpoint.js'></script>
-    <script src='../wwi/selector.js'></script>
-    <script src='../wwi/mouse_events.js'></script>
-    <script src='../wwi/multimedia_client.js'></script>
-    <script src='../wwi/texture_loader.js'></script>
-    <script src='../wwi/default_url.js'></script>
-    <script src='../wwi/server.js'></script>
-    <script src='../wwi/stream.js'></script>
-    <script src='../wwi/x3d.js'></script>
-    <script src='../wwi/x3d_scene.js'></script>
-    <script src='../wwi/animation.js'></script>
-    <script src='../wwi/webots.js'></script>
-    <script src='../wwi/webots.VideoStream.js'></script>
-    <script src='../wwi/node_modules/@babel/polyfill/dist/polyfill.js'></script>
-    <script src="setup_viewer.js"></script>
   </head>
+
   <body>
     <div style="padding-left:10px;height:100px">
       <h1>Webots streaming viewer</h1>
-      <p>
-        Connect to:
-        <input id="IPInput" type="text" value="localhost" />
-        :
-        <input id="PortInput" type="text" value="1234" maxlength="5" size="5"/>
-        mode: <select id="mode">
+      <p> Connect to: <input id="IPInput" type="text" value="localhost" /> : <input id="PortInput" type="text" value="1234" maxlength="5" size="5" /> mode: <select id="mode">
```

Comment:
Is this alignment dome automatically?
```suggestion
      <p> Connect to: <input id="IPInput" type="text" value="localhost" /> : <input id="PortInput" type="text" value="1234" maxlength="5" size="5" /> mode:
        <select id="mode">
```
If not this is quite strange and I would move the `<select>` tag on a new line so that is easily visible where does the `</select>` close tag come from.